dffml/accuracy/clf.py

```diff
@@ -21,16 +21,18 @@ class ClassificationAccuracyContext(AccuracyContext):
     Classification Accuracy
     """
 
-    async def score(self, mctx: ModelContext, sources: SourcesContext):
-        if len([mctx.parent.config.predict]) != 1:
+    async def score(
+        self, mctx: ModelContext, sources: SourcesContext, feature: Feature,
+    ):
+        if len([feature]) != 1 or isinstance(feature, Features):
             raise InvalidNumberOfFeaturesError(
-                f"{self.__class__.__qualname__} can only assess accuracy of one feature. features: {features}"
+                f"{self.__class__.__qualname__} can only assess accuracy of one feature. Refer to docs to find a scorer that supports Multi-Output"
             )
```

Comment:
We really don't need this check logic if we call with list argument expansion (`*features`) from `high_level.ml:accuracy()`.